src/client.ts

```diff
@@ -1101,19 +1102,28 @@ export class Client<Ctx = null> {
       } else if ('code' in event) {
         const closeEvent = <CloseEvent>event;
         if (closeEvent.code === CloseCode.POLICY_VIOLATION) {
-          // This means that the token was rejected. We need to fetch another one.
+          // This means that the token was rejected. Even though this is a
+          // permanent error from the perspective of the infrastructure, most
+          // of the time it can be corrected by fetching a new token, since
+          // this happens during cluster transfers most of the time (although
+          // sometimes it also happens when a particular Repl is tombstoned).
+          // TODO: Make the tombstone case return an USER_ERROR to have more
+          // clarity about why the Repl couldn't run.
```

Comment:
```suggestion
          // sometimes it also happens when a particular Repl is taken down).
          // TODO: Make the takedown case return an USER_ERROR to have more
          // clarity about why the Repl couldn't run.
```

nit: just to differentiate between tombstone for cluster transfer and tombstone for takedown